src/ViewportInputsStore.js

```diff
@@ -95,6 +92,26 @@ RootStore = types
         function setBeginEndBin(newBeginBin, newEndBin) {
             self.beginEndBin = [newBeginBin, newEndBin];
         }
+        function getPath() {
+            return self.pathNucPos.path;
+        }
+        function getNucPos() {
+            return self.pathNucPos.nucPos;
+        }
+        function getPathNucPos() {
+            return self.pathNucPos;
+        }
+        function updatePathNucPos(path, nucPos) {
```

Comment:
I am just curious, but what happens if the nucPos is greater then the path length?

Reply:
This is not really fail save.... we don't know the actual length of a path in the VIZ.
But if the users jumps to a path or a nucleotide position which are not in the index, the server will answer with `0` and an alert tells the user, that the requested path + position is not in the index.

Reply:
I see, that's fine.